gitattributes.go

```diff
@@ -0,0 +1,251 @@
+package enry
+
+import (
+	"errors"
+	"fmt"
+	"io"
+	"io/ioutil"
+	"regexp"
+	"strings"
+
+	"gopkg.in/src-d/enry.v1/data"
+)
+
+type attrType int
+
+const (
+	vendor attrType = iota
+	documentation
+	generated
+	language
+)
+
+const _attrType_name = "vendordocumentationgeneratedlanguage"
```

Comment:
This is autogenerated code by the command tool stringer its the most simple way to do a toString when you got an enum  [stringer](https://godoc.org/golang.org/x/tools/cmd/stringer)